src/App.jsx

```diff
@@ -1,14 +1,86 @@
+import { useState } from 'react';
 import './App.css';
+import ChatLog from './components/ChatLog';
+import messageData from './data/messages.json';
 
 const App = () => {
+  const [messages, setMessages] = useState(messageData);
+  const [likedCount, setLikedCount] = useState(0);
```

Comment:
Do we necessarily need this piece of state? Couldn't we just use the fact that if a message is liked then we could have a function that calculates the total number of likes by iterating over the messages that is `chatData`? This works each render since each time a message is liked `chatData` state is changed.